scripts/tests/browser_translation_test.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-undef */
+/* eslint-disable max-lines-per-function */
+
+
+ const baseURL = getRootDirectory(gTestPath).replace(
+   "chrome://mochitests/content",
+   "https://example.com"
+ );
+
+ add_task(async function testTranslationBarDisplayed() {
+   info("Test the Translation functionality");
+
+   // open the test page.
+   let tab = await BrowserTestUtils.openNewForegroundTab(
+     gBrowser,
+     `${baseURL }browser_translation_test.html`
+   );
+
+   /*
+    * the infobar is not triggered first time the page is loaded due some caching
+    * so we need to reload the tab in order to have it summoned
+    */
```

Comment:
> the infobar is not triggered first time the page is loaded due some caching

Is this a bug that we need to resolve later?

Reply:
This happens only in the mochistest framework. We fixed something similar in the production xpi here https://github.com/mozilla/firefox-translations/issues/69 though